ICMP Packet routing (#30)
quicwg/load-balancers: draft-ietf-quic-load-balancers.md

The server then encrypts the second through seventeenth octets using the 128-bit
AES-ECB cipher.

# ICMP Processing

For protocols where 4-tuple load balancing is sufficient, it is straightforward

> to deliver ICMP packets from the network to the server that can consume it, by

Reviewer: ```suggestion
to deliver ICMP packets from the network to the correct server, by
```
